docs/man_pages/screenbuilder/screenbuilder.md

```diff
@@ -28,4 +28,11 @@ Command | Description
 Command | Description
 ----------|----------
 [create screenbuilder](../project/creation/create-screenbuilder.html) | Creates a new project for hybrid development with Screen Builder.
+[add-about](add-about.html) | Inserts an about form in an existing application view.
```

Comment:
This seems redundant here =) After all we have the exact same list a few lines above.